features/step_definitions/docker_steps.rb

```diff
@@ -90,7 +90,7 @@ def retrieve_minion_id
   visit("https://#{$server.full_hostname}/rhn/manager/cm/build")
 end
 
-And(/^I verify that all "([^"]*)" container images were built correctly in the gui$/) do |count|
+And(/^all "([^"]*)" container images should be built correctly in the GUI$/) do |count|
```

Comment:
Here should be All --> big A

Reply:
Why not. English would command lower case after "then" or "and", but we can make it a convention

Reply:
I tried, it looks ugly with upper case